File: specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/examples-js/Secrets_Get.js
const { CdnManagementClient } = require("@azure/arm-cdn");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets an existing Secret within a profile.
 *
 * @summary Gets an existing Secret within a profile.
 * x-ms-original-file: specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/examples/Secrets_Get.json
 */
async function secretsGet() {
  const subscriptionId = "subid";
  const resourceGroupName = "RG";
  const profileName = "profile1";
  const secretName = "secret1";
  const credential = new DefaultAzureCredential();
  const client = new CdnManagementClient(credential, subscriptionId);
  const result = await client.secrets.get(resourceGroupName, profileName, secretName);
  console.log(result);
}

secretsGet().catch(console.error);

File: specification/cdn/resource-manager/Microsoft.Cdn/stable/2024-02-01/examples-java/Secrets_Get.java

/**
 * Samples for Secrets Get.
 */
public final class Main {
    /*
     * x-ms-original-file: specification/cdn/resource-manager/Microsoft.Cdn/stable/2024-02-01/examples/Secrets_Get.json
     */
    /**
     * Sample code: Secrets_Get.
     * 
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void secretsGet(com.azure.resourcemanager.AzureResourceManager azure) {
        azure.cdnProfiles().manager().serviceClient().getSecrets().getWithResponse("RG", "profile1", "secret1",
            com.azure.core.util.Context.NONE);
    }
}
